config/cmake/buildSundials.cmake

```diff
@@ -46,7 +46,7 @@ ExternalProject_Add(sundials
         -DEXAMPLES_INSTALL=OFF
 		-DSUNDIALS_INDEX_TYPE=int32_t
         -DKLU_ENABLE=ON
-        "-DCMAKE_C_FLAGS=${EXTRA_C_FLAGS}"
```

Comment:
Based on where EXTRA_C_FLAGS is set right now, it shouldn't have spaces in it. I think the change was because putting it in quotes was resulting in `;-DCMAKE_C_FLAGS=${EXTRA_C_FLAGS};` appearing in the generated CMakeLists.txt file.

Reply:
Okay, got it. It just needed escaped quotes.